tools/reviewer/webapp/src/app/dashboard/diff/diff.service.ts

```diff
@@ -1,15 +1,31 @@
 import { Injectable } from '@angular/core';
-import { Router } from '@angular/router';
+import { Params, Router } from '@angular/router';
 
 import { File } from '@/core/proto';
 
 @Injectable()
 export class DiffService {
   constructor(private router: Router) { }
 
-  openFile(file: File, diffId: number, commitId?: string): void {
-    this.router.navigate([
-      'diff/' + diffId + '/' + file.getFilenameWithRepo(),
-    ], { queryParams: { right: commitId } });
+  openFile(
+    isNewTab: boolean,
+    file: File,
+    diffId: number,
+    commitId?: string,
+  ): void {
+    const url: string = 'diff/' + diffId + '/' + file.getFilenameWithRepo();
+    const queryParams: Params = { right: commitId };
```

Comment:
Looks like you only need `const queryParams: Params = { right: commitId };` in the `else`.